src/main/java/bio/terra/service/filedata/exception/MaxDrsLookupsException.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.service.filedata.exception;
+
+import bio.terra.common.exception.BadRequestException;
+
+public class MaxDrsLookupsException extends BadRequestException {
```

Comment:
Ah, that's tried to my earlier comment.  I think you want to create a new `TooManyRequestsException` and this class should extend that